### PR TITLE
Small improvements in area chart pages

### PR DESCRIPTION
**Pull request by arni-magnusson**
Hi, thanks for the nicely organized collection of plot snippets. This pull request makes small improvements in the 'area chart' pages, mainly fixing typos, links, and such.

### CODE REVIEW

File: 165-basic-area-chart.Rmd [hunk 1]
```diff
@@ -30,14 +30,14 @@ knitr::opts_chunk$set( warning=FALSE, message=FALSE)
 
 
 
-# Basic line chart with `ggplot2` and `geom_line()`
```
Comment (holtzy) [inline 165-basic-area-chart.Rmd:33]: OMG thanks 🙈

File: 136-stacked-area-chart.Rmd [hunk 2]
```diff
@@ -30,7 +30,7 @@ knitr::opts_chunk$set( warning=FALSE, message=FALSE)
 
 
 
-# Most basic stacked area with ggplot2
```
Comment (holtzy) [inline 136-stacked-area-chart.Rmd:33]: I like it when examples are _really_ simple at the beginning of a tutorial. Does _Most basic_ sound weird here? 🤔 (I'm not native)

**Review by holtzy — COMMENTED** [review]
Thanks so much for your PR! 🙏

There is just one chunk that I would like to keep please. Otherwise I'm happy to merge it.

File: 136-stacked-area-chart.Rmd (hunk 2 above)
Comment (arni-magnusson) [reply 136-stacked-area-chart.Rmd:33]: I agree that simple examples are great, and simple heading are also great :) The reader expects the first example to be simple and we can let them dive straight into the code.

**Comment by arni-magnusson** [comment]
Thanks for the positive and quick response; I put the chunk back in. It was a copy-paste from the 'Note on area chart' so I renamed it to 'Note on stacked area chart' :)

**Comment by arni-magnusson** [comment]
> Thanks so much for your PR!
> 
> There is just one chunk that I would like to keep please. Otherwise I'm happy to merge it.

Great!

With commit https://github.com/arni-magnusson/R-graph-gallery/commit/6fd07e9 the chunk has been added back in the pull request, I agree it's good to keep that chunk.

Looking forward to the merge :)

**Comment by holtzy** [comment]
Thank you so much.
I am very sorry it took me so long to merge. I am starting to work actively on the gallery just now.

**Pull request merged**